src/main/io/osd.c

```diff
@@ -460,6 +462,36 @@ static bool osdDrawSingleElement(uint8_t item)
     char buff[OSD_ELEMENT_BUFFER_LENGTH] = "";
 
     switch (item) {
+    case OSD_ARROW_TURTLE_MODE: 
+        {
+            const int angleR = attitude.values.roll;
+            const int angleP = attitude.values.pitch; // still gotta update all angleR and angleP pointers.
+            if (IS_RC_MODE_ACTIVE(BOXFLIPOVERAFTERCRASH)) {
+                if (angleP > 0 && ((angleR > 175 && angleR < 180) || (angleR > -180 && angleR < -175))) {
+				buff[0] = SYM_ARROW_SOUTH;
```

Comment:
Indentation. Please make sure that you are using 4 spaces instead of tabs for indentation, and adjust your editor settings accordingly.